### DISCUSSION TITLE
[Question] How to integrate .xcframework SDK?

### DISCUSSION THREAD

**Discussion by a92c** [NativeScript/NativeScript#9975] [Q&A] [answered]
### Is your feature request related to a problem? Please describe.

Hi, how do I integrate a native ios sdk, as described here: https://nstudio.io/blog/manual-ios-framework-integrations, with this new version of the seed?
I don't know if I'm on the right path, the sdk in my possession is in xcframework format.
Thanks for your help.

https://github.com/NativeScript/plugin-seed/issues/28 

### Please accept these terms

- [X] I have searched the [existing issues](https://github.com/NativeScript/NativeScript/issues) as well as [StackOverflow](https://stackoverflow.com/questions/tagged/nativescript) and this has not been posted before
- [X] I agree to follow this project's [Code of Conduct](https://github.com/NativeScript/NativeScript/blob/master/tools/notes/CONTRIBUTING.md#coc)

**Comment by edusperoni** [comment, marked as answer]
Hey @a92c!

You could just drop it in the platforms/ios folders. You can see an example here: https://github.com/NativeScript/plugins/tree/main/packages%2Fios-security

**Comment by NathanWalker** [comment]
@a92c you may also find this helpful for further context/detail; same thing applies to .xcframework's and .framework's:
https://nstudio.io/blog/manual-ios-framework-integrations

**Comment by PixsaOJ** [comment]
Has the method been updated for this? Manually dropping in auto generated folder feels unsafe. 

Also, is this discussion about plugins? I am just trying to use  framework in my project which has same structure as provided example
